Cargo.lock

```diff
@@ -215,13 +215,13 @@ checksum = "9b34d609dfbaf33d6889b2b7106d3ca345eacad44200913df5ba02bfd31d2ba9"
 
 [[package]]
 name = "async-trait"
-version = "0.1.71"
+version = "0.1.72"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "a564d521dd56509c4c47480d00b80ee55f7e385ae48db5744c67ad50c92d2ebf"
+checksum = "cc6dde6e4ed435a4c1ee4e73592f5ba9da2151af10076cc04858746af9352d09"
```

Comment:
There are quite a few checksum updates in this commit, but I don't see why there would be based on your changes..

Reply:
My guess why this happened is because I removed `toml=0.5` and each library that is updated here was bounded by that versioning. So i did not run cargo update or something.